### ISSUE TITLE
Notifications settings text not visible to screen readers

### ISSUE THREAD

**Issue by FediVideos** [Chocobozzz/PeerTube#6049]
### Describe the current behavior

Just chatting with a blind PeerTube user, they are unable to set their notifications because the text labels next to the toggle switches are not visible to their screen reader app. (Possibly a JavaScript problem?)

@rkingett can give more advice on screen reader they are using.

### Steps to reproduce

1. Go to My Account > Notifications > Notification Preferences
2. Read screen using screen reader software
3. Attempt to read labels next to notification toggle switches

### Describe the expected behavior

The text labels which are next to toggle switches ought to be visible to the screen reader.

### Additional information

* PeerTube instance:
  * URL: https://spectra.video
  * Version: 6.0.0-rc.1
  * NodeJS version:
  * Ffmpeg version:

* Browser name, version and platforms on which you could reproduce the bug:
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

**Comment by rkingett** [comment]
For reference, I am using [the NVDA screen reader on Windows,](https://www.nvaccess.org/download/) running the latest version of FireFox.

In particular, this is the page inside of the settings dashboard. I am using [this  instance that has an RC version of PeerTube](https://spectra.video/about) so I am unsure if that makes a difference.

**Comment by Chocobozzz** [comment]
Thanks! This should be fixed by https://github.com/Chocobozzz/PeerTube/commit/05d6520b22f8c60bff8894080b5c727270474f73

Don't hesitate to create other issues if you notice problems with a screen reader on PeerTube :)
